shell/common/node_bindings.h

```diff
@@ -162,13 +160,21 @@ class NodeBindings {
   // Which environment we are running.
   const BrowserEnvironment browser_env_;
 
-  // Current thread's MessageLoop.
-  scoped_refptr<base::SingleThreadTaskRunner> task_runner_;
+  // Loop used when constructed in WORKER mode
+  uv_loop_t worker_loop_;
 
   // Current thread's libuv loop.
-  raw_ptr<uv_loop_t> uv_loop_;
+  // depends-on: worker_loop_
+  const raw_ptr<uv_loop_t> uv_loop_;
```

Comment:
Can `uv_loop_`, `worker_loop_` and `task_runner_` be private members ? I don't remember why they were protected in the first place.

Reply:
Yeah, that was kind of a compromise. All three [mac, win, linux] subclasses access `uv_loop_` directly, so it needs to be protected. And that meant `worker_loop_` needed to be protected too unless we are going to co-mingle private and protected instead of keeping the sorted public -> protected -> private ordering :man_shrugging: 

*However* there's also a uv_loop getter. If I update the subclasses to use that, then everything can be private...

Reply:
Revised in  c463b77a608a9d51cdbee3ed0f31ae2edd91e0b6